scenarios/sumo/intersections/1_to_2lane_left_turn_t_agents_1/scenario.py

```diff
@@ -23,65 +23,78 @@
 from pathlib import Path
 
 from smarts.sstudio import gen_scenario
-from smarts.sstudio.types import Flow, Mission, Route, Scenario, Traffic, TrafficActor
+from smarts.sstudio.types import (
+    Flow,
+    Mission,
+    Route,
+    Scenario,
+    Traffic,
+    TrafficActor,
+    TrapEntryTactic,
+)
 
 normal = TrafficActor(
     name="car",
 )
 
-horizontal_routes = [
-    ("E4", 0, "E1", 0),
-    ("E4", 1, "E1", 1),
-    ("-E1", 0, "-E4", 0),
-    ("-E1", 1, "-E4", 1),
-]
-
-turn_left_routes = [
-    ("E0", 0, "E1", 1),
-    ("E4", 1, "-E0", 0),
+# flow_name = (start_lane, end_lane)
+route_opt = [
+    (0, 0),
+    (1, 1),
+    (2, 2),
 ]
 
-turn_right_routes = [
-    ("E0", 0, "-E4", 0),
-    ("-E1", 0, "-E0", 0),
-]
-
-# Total route combinations = 8C1 + 8C2 + 8C3 + 8C4 + 8C5 = 218
-# Repeated route combinations = 218 * 2 = 436
-all_routes = horizontal_routes + turn_left_routes + turn_right_routes
+# Traffic combinations = 3C2 + 3C3 = 3 + 1 = 4
+# Repeated traffic combinations = 4 * 100 = 400
+min_flows = 2
+max_flows = 3
 route_comb = [
-    com for elems in range(1, 6) for com in combinations(all_routes, elems)
-] * 2
+    com
+    for elems in range(min_flows, max_flows + 1)
+    for com in combinations(route_opt, elems)
+] * 100
+
 traffic = {}
 for name, routes in enumerate(route_comb):
     traffic[str(name)] = Traffic(
         flows=[
             Flow(
                 route=Route(
-                    begin=(start_edge, start_lane, 0),
-                    end=(end_edge, end_lane, "max"),
+                    begin=("gneE3", start_lane, 0),
+                    end=("gneE4", end_lane, "max"),
```

Comment:
This map `scenarios/sumo/intersections/1_to_2lane_left_turn_t_agents_1/scenario.py` seems to only contain `E0`, `E1`, and `E4` edges. What does `gneE3`, `gneE4`, and `gneE6` refer to?

Reply:
I think this file was changed and somehow did not get updated when I used rebase. I am going to fix it.